Automattic/vip-workflow-plugin
# Update/blueprint

## Description

A few sentences describing the overall goals of the Pull Request.

Should include any special considerations, decisions, and links to relevant GitHub issues.

Please don't include internal or private links :)

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
2. Add a pitch titled "why cookies are so delicious".
3. Verify post is also delicious.
